src/coreclr/jit/lowerxarch.cpp

```diff
@@ -2506,8 +2622,65 @@ GenTree* Lowering::LowerHWIntrinsicCreate(GenTreeHWIntrinsic* node)
     //          +--*  ... T
     //          +--*  opN T
     //   node = *  HWINTRINSIC   simd   T Create
+    if (intrinsicId == NI_Vector512_Create)
+    {
+        assert(comp->compIsaSupportedDebugOnly(InstructionSet_AVX2));
 
-    if (intrinsicId == NI_Vector256_Create)
+        // We will be constructing the following parts:
+        //          /--*  op1 T
+        //          +--*  ... T
+        //   lo   = *  HWINTRINSIC   simd32 T Create
+        //          /--*  ... T
+        //          +--*  opN T
+        //   hi   = *  HWINTRINSIC   simd32 T Create
+        //   idx  =    CNS_INT       int    1
+        //          /--*  lo   simd64
+        //          +--*  hi   simd32
+        //          +--*  idx  int
+        //   node = *  HWINTRINSIC   simd64 T InsertVector256
+
+        // This is roughly the following managed code:
+        //   ...
+        //   var lo   = Vector256.Create(op1, ...);
+        //   var hi   = Vector256.Create(..., opN);
+        //   return Avx2.InsertVector512F(lo, hi, 0x01);
```

Comment:
Wouldn't this be `Vector512.Create(lo, hi)` or `Avx512F.InsertVector256(lo.ToVector512Unsafe(), hi, 1)`

There is no `Avx2.InsertVector512F` since `vinsertf64x4` is only available on AVX512F+

Reply:
Done. Was a spelling error in comments